src/orion/benchmark/assessment/averagerank.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.benchmark.assessment` -- Average Rank Assessment
+================================================================
+
+.. module:: assessment
+   :platform: Unix
+   :synopsis: Benchmark algorithms with average rank.
+
+"""
+
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+from orion.benchmark.base import BaseAssess
+
+
+class AverageRank(BaseAssess):
+    """
+    For each algorithm, run fixed number of Experiment, average the rank of trials for
```

Comment:
Sorry, I am unable to understand the description here. Could you please try to rephrase it? :sweat_smile:

Reply:
New version for reference:

```diff
- For each algorithm, run fixed number of Experiment, average the rank of trials for
- the same algorithm at the same trial sequence order.
- For the performance of trials in an Experiment, instead using the actual trial objective value,
- here we use the best objective value in the same Experiment until the particular trial.
+ Evaluate the average performance (objective value) between different search algorithms from
+ the rank perspective at different time steps (trial number).
+ The performance (objective value) used for a trial will the best result until the trial.
```

I think it is much more clear, I only had to read it once to understand. :) Thanks!